internal/fs/vss_windows.go

```diff
@@ -810,6 +810,17 @@ func initializeVssCOMInterface() (*ole.IUnknown, error) {
 		}
 	}
 
+	// initialize COM security for VSS, this can't be called more then once
+	if err = ole.CoInitializeSecurity(
+		-1,   // Allow *all* VSS writers to communicate back!
+		5,    // RPC_C_AUTHN_LEVEL_PKT_INTEGRITY
+		3,    // RPC_C_IMP_LEVEL_IMPERSONATE
+		0x40, // EOAC_DYNAMIC_CLOAKING
```

Comment:
According to [the documentation](https://learn.microsoft.com/en-us/windows/win32/vss/security-considerations-for-requestors#setting-requester-default-com-access-check-permission), this should be with the options (from Windows 8 onwards):
 - `RPC_C_AUTHN_LEVEL_PKT_PRIVACY`
 - `RPC_C_IMP_LEVEL_IMPERSONATE`
 - EOAC_STATIC

Reply:
[EOLE_AUTHENTICATION_CAPABILITIES](https://learn.microsoft.com/en-us/windows/win32/api/objidl/ne-objidl-eole_authentication_capabilities) does not have a declaration for `EOAC_STATIC`, there only exists `EOAC_STATIC_CLOAKING` could be the document is not complete or did they mean `EOAC_STATIC_CLOAKING`?

_enumeration from objidl.h_
```cpp
typedef enum tagEOLE_AUTHENTICATION_CAPABILITIES {
    EOAC_NONE = 0x0,
    EOAC_MUTUAL_AUTH = 0x1,
    EOAC_STATIC_CLOAKING = 0x20,
    EOAC_DYNAMIC_CLOAKING = 0x40,
    EOAC_ANY_AUTHORITY = 0x80,
    EOAC_MAKE_FULLSIC = 0x100,
    EOAC_DEFAULT = 0x800,
    EOAC_SECURE_REFS = 0x2,
    EOAC_ACCESS_CONTROL = 0x4,
    EOAC_APPID = 0x8,
    EOAC_DYNAMIC = 0x10,
    EOAC_REQUIRE_FULLSIC = 0x200,
    EOAC_AUTO_IMPERSONATE = 0x400,
    EOAC_NO_CUSTOM_MARSHAL = 0x2000,
    EOAC_DISABLE_AAA = 0x1000
} EOLE_AUTHENTICATION_CAPABILITIES;
```